### PR TITLE
Create Psammead locale with Igbo & Pidgin Moment Locale

### LINKED ISSUE #553: Create psammead package for moment locales
Jira - WSFRONTPGS-65

**Is your feature request related to a problem? Please describe.**
For WS frontpages we need localised timestamps for Pigin, Igbo and Yoruba. Luckily [moment.js](http://momentjs.com) already supports Yoruba, but we will need to create local configs for Igbo and Pidgin

**Describe the solution you'd like**
Creat a package, name TBC, that contains 2 files, in the format of [existing configs](https://github.com/moment/moment/blob/develop/locale/yo.js)

When you pick this up, as around for what the package should be called

Eventually these could be contributed back to Moment, but for now this is nice place to keep them!

**Testing notes**
[Tester to complete]

Dev insight: Could we have a psammead story which pulls in moment, loads these locales and then uses them to make a bunch of timestamp formats? Unit tests too

### PR DESCRIPTION
**Pull request by dr3**
Resolves #553

**Overall change:** Adds moment locales for igbo and pidgin

**Code changes:**

- Adds moment locales, in the official style
- Adds moment locale tests, in the official style

---

- [x] I have assigned myself to this PR and the corresponding issues
- ] Tests added for new features
- [x] Test engineer approval

### CODE REVIEW

**Review by jroebu14 — APPROVED** [review]
LGTM

**Review by j-pendlebury — APPROVED** [review]
Looks good 👍

File: packages/utilities/psammead-locales/moment/ig.js [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+  weekdays: 'Sọnde_Mọnde_Tuzde_Wenesde_Tọsde_Fraịde_Satọde'.split('_'),
+  weekdaysShort: 'Sọn_Mọn_Tuz_We_Tọs_Frai_Sat'.split('_'),
+  weekdaysMin: 'Su_Mọ_Tu_We_Tọ_Fr_Sa'.split('_'),
+  longDateFormat: {
+    LT: 'HH:mm',
+    LTS: 'HH:mm:ss',
+    L: 'DD/MM/YYYY',
+    LL: 'D MMMM YYYY',
+    LLL: 'D MMMM YYYY HH:mm',
+    LLLL: 'dddd, D MMMM YYYY HH:mm',
```
Comment (DenisHdz) [inline packages/utilities/psammead-locales/moment/ig.js]: I believe this is as follows
```suggestion
    LLLL: 'dddd, MMMM D YYYY HH:mm',
```

File: packages/utilities/psammead-locales/moment/ig.js [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+    d: 'otu ụbọchị',
+    dd: 'Ụbọchị %d',
+    M: 'otu ọnwa',
+    MM: 'Ọnwa %d',
+    y: 'Otu afọ',
+    yy: 'Afọ %d',
+  },
+  dayOfMonthOrdinalParse: /(Nke )\d{1,2}/,
+  ordinal: function ordinal(number) {
+    return 'Nke '.concat(number);
```
Comment (andrew-nowak) [inline packages/utilities/psammead-locales/moment/ig.js:45]: Are we sure about this? The full examples given in the translations document don't feature this word (eg. "4 Maach 2019 14:25")

**Review by andrew-nowak — CHANGES_REQUESTED** [review]
Also https://github.com/bbc/psammead/pull/638#discussion_r294296109

File: packages/utilities/psammead-locales/moment/ig.js (hunk 2 above)
Comment (dr3) [reply packages/utilities/psammead-locales/moment/ig.js:45]: Format `LLLL` becomes `Sọnde, 14 Febụwarị 2010 15:25` so I think it's ok? what cha think?

Comment (dr3) [reply packages/utilities/psammead-locales/moment/ig.js:45]: the ordinal I think its like `1st March` vs `1 March`, the `1st/Nke 1` logic I think is fine, just up to us in the various date formats to use it or not like `L, LL, LLL` etc

Comment (andrew-nowak) [reply packages/utilities/psammead-locales/moment/ig.js:45]: Ah I understand now, this makes sense. Does this mean the pidgin LL, LLL, LLLL should use Do to get ordinals then?

File: packages/utilities/psammead-locales/moment/pcm.js [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+  ),
+  weekdaysShort: 'Sun_Mon_Tue_Wed_Thu_Fri_Sat'.split('_'),
+  weekdaysMin: 'Su_Mo_Tu_We_Th_Fr_Sa'.split('_'),
+  longDateFormat: {
+    LT: 'HH:mm',
+    LTS: 'HH:mm:ss',
+    L: 'DD/MM/YYYY',
+    LL: 'D MMMM YYYY',
+    LLL: 'D MMMM YYYY HH:mm',
+    LLLL: 'dddd, D MMMM YYYY HH:mm',
```
Comment (andrew-nowak) [inline packages/utilities/psammead-locales/moment/pcm.js]: Translations doc makes it look like this should be the same as LLL?

Comment (dr3) [reply packages/utilities/psammead-locales/moment/pcm.js]: Which line? Can you copy the example please :P

Comment (andrew-nowak) [reply packages/utilities/psammead-locales/moment/pcm.js]: LLL: 3rd March 2019 14:25
LLLL: 3rd March 2019 14:25
lines 97 & 98

File: packages/utilities/psammead-locales/moment/ig.js (hunk 2 above)
Comment (dr3) [reply packages/utilities/psammead-locales/moment/ig.js]: Ah yes, done :)

Comment (dr3) [reply packages/utilities/psammead-locales/moment/ig.js:45]: Ah yes, done :)

File: packages/utilities/psammead-locales/moment/pcm.js (hunk 3 above)
Comment (dr3) [reply packages/utilities/psammead-locales/moment/pcm.js]: Ahh good spot, done :)

File: packages/utilities/psammead-locales/moment/ig.js [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+    future: 'na %s',
+    past: '%s gara aga',
+    s: 'ntabịanya ole na ole',
+    ss: 'ntabịanya %d',
+    m: 'otu nkeji',
+    mm: 'nkeji %d',
+    h: 'otu elekere',
+    hh: 'elekere %d',
+    d: 'otu ụbọchị',
+    dd: 'Ụbọchị %d',
```
Comment (andrew-nowak) [inline packages/utilities/psammead-locales/moment/ig.js:37]: I can see these are capitalised as in the document, but it also doesn't look right. `Ụbọchị` isn't capitalised in the line above, for example. I'm not sure if that's something we can query, or how/in what circumstances moment will cap/uncap these strings anyway.

**Review by DenisHdz — APPROVED** [review]
LGTM 💅🏼

File: packages/utilities/psammead-locales/moment/ig.js (hunk 4 above)
Comment (andrew-nowak) [reply packages/utilities/psammead-locales/moment/ig.js:37]: Opening the storybook page for igbo, the date with ordinals is `Juun Nke 17 2019, 3:16:15 pm`, the capitalised `Nke` definitely looks odd.

**Review by andrew-nowak — APPROVED** [review]
I'm unsure about the correctness of some of the capitalisations, but they're as they are in the translations document and the code looks good, so I'll approve this PR. If there's any avenue for querying those capitalisations, I think we should

File: packages/utilities/psammead-locales/moment/ig.js [hunk 5]
```diff
@@ -0,0 +1,51 @@
...
+    LL: 'D MMMM YYYY',
+    LLL: 'D MMMM YYYY HH:mm',
+    LLLL: 'dddd, MMMM D YYYY HH:mm',
+  },
+  calendar: {
+    sameDay: '[Taa na] LT',
+    nextDay: '[Echi na] LT',
+    nextWeek: 'dddd [na] LT',
+    lastDay: '[Unyaahụ na] LT',
+    lastWeek: 'dddd [gara aga na] LT',
```
Comment (david-boydell) [inline packages/utilities/psammead-locales/moment/ig.js:24]: Hi @dr3, are the above two values what we intend? The Google Sheet has:

```
[Unyaahụ] [na] 12:34
Tuzde [gara aga] [na] 12:34
```
Wasn't sure if we were missing some square brackets or not.

Comment (dr3) [reply packages/utilities/psammead-locales/moment/ig.js:24]: Yeah the brackets can be joined up

**Comment by david-boydell** [comment]
LGTM, this is ready for merge.

**Pull request merged**